src/mumble/TextToSpeech.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2005-2017 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "mumble_pch.hpp"
+
+#include "TextToSpeech.h"
+
+#include <QTextToSpeech>
+
+class TextToSpeechPrivate {
+	public:
+		QTextToSpeech *mSpeech;
+		QVector<QVoice> mVoices;
+		TextToSpeechPrivate();
+		~TextToSpeechPrivate();
+		void say(const QString &text);
+		void setVolume(int v);
+};
+
+TextToSpeechPrivate::TextToSpeechPrivate() {
+	mSpeech = NULL;
+	mSpeech = new QTextToSpeech(this);
+}
+
+TextToSpeechPrivate::~TextToSpeechPrivate() {
+	if (mSpeech)
+		delete mSpeech;
```

Comment:
You should not need to delete mSpeech, since it is a QObject and we're its parent.
If we are destroyed, our children will be destroyed, too.

(Leave an empty destructor)